etc/kayobe/kolla-image-tags.yml

```diff
@@ -42,3 +42,6 @@ kolla_image_tags:
   ironic_neutron_agent:
     rocky-9: 2024.1-rocky-9-20241023T143407
     ubuntu-jammy: 2024.1-ubuntu-jammy-20241023T143407
+  opensearch:
```

Comment:
Could you move this block below to Octavia for alphabetical order?

Reply:
No.


Yeah, sure!